static/views/LockDeploy.jsx

```diff
@@ -0,0 +1,122 @@
+import {useCallback, useEffect, useMemo, useState} from 'react';
+import {useLocation, useNavigate, useOutletContext} from 'react-router-dom';
+
+import useApi from 'app/hooks/useApi';
+import useLocalStorage from 'app/hooks/useLocalStorage';
+
+function LockDeploy() {
```

Comment:
nit: fat arrow?

I know this is a whole debate too and here they'll be functionally the same 🤷 
feels like this should be tsc discussion if it hasn't been already though

Reply:
indifferent also 🤷